web/gds-admin-ui/src/pages/app/lists/index.js

```diff
@@ -128,11 +132,11 @@ const VaspsList = (): React$Element<React$FragmentType> => {
                                 </Col>
                             </Row>
                             {
-                                vasps && vasps.length &&
+                                data && data.vasps && data.vasps.length &&
                                 <Table
                                     columns={columns}
-                                    data={vasps}
-                                    pageSize={5}
+                                    data={data?.vasps}
+                                    pageSize={data.page_size || 100}
```

Comment:
So does the data table map to the page size; meaning as you paginate in the table you're making requests to the server? I guess I'm not familiar with how redux and the API are interacting here.

Reply:
Yes, it does. For the page size I don't make request currently

Reply:
I take all the data and paginate it on my side

Reply:
But I'm going to fix it soon